.github/workflows/README.md

```diff
@@ -53,11 +63,27 @@ Some actions require authentication to GCP to push container images. These worke
 - builds and pushes images to official GCR repo tagged with git commit
 - builds and pushes images to official GCR repo tagged as latest
 
-### push-tags.yaml
+
+### Push-Tags.yaml
 
 #### Triggers
 - tags pushed to repo
 
 #### Actions
 - builds and pushes images to official GCR repo tagged with git tag name
 
+
+### E2E.yaml
+
+#### Triggers
+- on each commit to master
+- on each commit to a release branch
+- on each PR into master
+
+#### Actions
+- ensure end-to-end test project has deleted all test resources
```

Comment:
The E2E test should run once or twice daily but not on each PR or each commit to master. If we run it on each PR, we'll need to handle multiple runs in parallel and also reduce velocity.

Reply:
I changed it to run on each commit to master, as we discussed in the meeting. I think that's a good first step, and if that's causing issues we can move it to periodically